calicoctl/commands/common/resources.go

```diff
@@ -20,6 +20,8 @@ import (
 	"fmt"
 	"os"
 
+	"github.com/projectcalico/calicoctl/calicoctl/commands/file"
```

Comment:
Does this need to be goimported? I see different grouping below?

Reply:
it was - but that only shifts things out of place it doesn't necessarily re-group things.  I've fixed.